tests/installation/zdup.pm

```diff
@@ -101,6 +102,25 @@ sub run() {
         assert_script_run("zypper -n addrepo \"$r\" repo$nr");
         $nr++;
     }
+    # resgiter modules before zdup migration
+    if (get_var('SCC_ADDONS', '') =~ /asmm|contm|hpcm|lgm|pcm|tcm|wsm|idu|ids/) {
+        save_screenshot;
+        my $repo_version = get_var("HDDVERSION");
+        set_var("ZDUP_MODULE", 1);
```

Comment:
where is this test variable being used? I guess it would be saver when we try to avoid writing to test variables and instead use internal variables when needed.